src/modules/FileLocksmith/FileLocksmithExt/resource.base.h

```diff
@@ -8,6 +8,7 @@
 #define FILE_DESCRIPTION "PowerToys File Locksmith Static Library"
 #define INTERNAL_NAME "PowerToys.FileLocksmithLib.lib"
 #define ORIGINAL_FILENAME "PowerToys.FileLocksmithLib.lib"
+#define IDI_FILELOCKSMITH 1001
```

Comment:
@stefansjfw
I changed this line after my local test from 1003 to 1001. Maybe we have a conflict. I will test it with 1001 on my local system again. If it doesn't work with 1001 I will switch back to 1003. (I am not so familiar with these things.)

Reply:
You can manually add these registry entries https://github.com/microsoft/PowerToys/blob/main/installer/PowerToysSetup/FileLocksmith.wxs#L19

Reply:
Actually, my bad. It's working. Probably Explorer cached previous one. After restarting explorer icon is there.

Reply:
Great news.